pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -18383,7 +18383,6 @@ self: {
        ];
        description = "Shell script analysis tool";
        license = lib.licenses.gpl3Only;
-       hydraPlatforms = lib.platforms.none;
```

Comment:
I did this manually, setting `doDistribute` and regenerating didn't seem to work.

Reply:
`doDistribute` doesn't affect the source, just the evaluation result, what exactly did not work?

Reply:
> `doDistribute` doesn't affect the source, just the evaluation result, what exactly did not work?

I thought `doDistribute` would remove `hydraPlatforms = lib.platforms.none;` but setting it and regenerating didn't produce any changes.

> Changes in `hackage-packages.nix` will be rolled back.

Yes, I know. What is the correct way to remove this line?

Reply:
> Yes, I know. What is the correct way to remove this line?

I think they are non-configurable for the versioned attributes, but `doDistribute` would achieve the same *effect* after evaluation. In any case populating `hydraPlatforms` for this is not strictly necessary, since it is not poisonous: If `shellcheck` has `hydraPlatforms != []` (or unspecified), `shellcheck` and all its dependencies (including `ShellCheck_0_8_0`) will be built regardless of their individual `hydraPlatforms` lists.

Reply:
`shellcheck` inherits `meta` from `ShellCheck_0_8_0` which will include the disabled `hydraPlatforms`.

Reply:
Pushed `doDistribute` for `ShellCheck_0_8_0`.